extensions/amp-ad-exit/0.1/config.js

```diff
@@ -37,7 +37,16 @@ export let AmpAdExitConfig;
 export let NavigationTargetConfig;
 
 /**
- * @typedef {!Object<string, {defaultValue: (string|number|boolean)}>}
+ * @typedef {{
+ *   defaultValue: (string|number|boolean),
+ *   vendorAnalyticsSource: (string|undefined),
+ *   vendorAnalyticsResponseKey: (string|undefined)
+ * }}
```

Comment:
Actually...I just played around with it and I think I would prefer for it to stay as-is. The other occurrence, VariableFrom3pAnalytics, is more restrictive than this, in that its keys must be strings. Without that, check-types is giving me stuff like the following (note that I renamed "vals"):
```
extensions/amp-ad-exit/0.1/amp-ad-exit.js:114: ERROR - restricted index type
found   : (string|undefined)
required: string
                  map[variableFrom3pAnalytics.vendorAnalyticsResponseKey]) {
```